lifetimes/fitters/base_fitter.py

```diff
@@ -0,0 +1,61 @@
+"""Base fitter for other classes."""
+import dill
+from ..utils import _save_obj_without_attr
+
+
+class BaseFitter(object):
+    """Base class for fitters."""
+
+    def __repr__(self):
+        """Representation of fitter."""
+        classname = self.__class__.__name__
+        try:
+            param_str = ", ".join("%s: %.2f" % (param, value) for param, value
+                                  in sorted(self.params_.items()))
+            return "<lifetimes.%s: fitted with %d subjects, %s>" % (
+                classname, self.data.shape[0], param_str)
```

Comment:
@stochastic1 brings up a good point - does repr still work if we don't save the data with it? Perhaps we need to protect against this case too.